src/EA.Weee.RequestHandlers.Tests.Unit/Admin/Reports/GetUkNonObligatedWeeeReceivedDataCsvHandlerTests.cs

```diff
@@ -0,0 +1,70 @@
+﻿namespace EA.Weee.RequestHandlers.Tests.Unit.Admin.Reports
+{
+    using System;
+    using System.Security;
+    using System.Threading.Tasks;
+    using Core.Admin;
```

Comment:
Can some tests be added to check the CSV, similar to that in GetAatfAeReturnDataCsvHandlerTests